### ISSUE TITLE
BUG: Page count is not correct for e-filed documents

### ISSUE THREAD

**Issue by klohman** [ustaxcourt/ef-cms#7376]
**Describe the Bug**
efiled documents (after the Petition) are showing "0" as page count. 

**Business Impact/Reason for Severity**

**In which environment did you see this bug?**
stg

**Who were you logged in as?**
flexionustc+petitioner@gmail.com

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
manual testing 

**To Reproduce**
1. Login as petitioner
2. Click into case
3. File a document
4. Click document link from docket record

**Expected Behavior**
Page count should match teh actual PDF + Coversheet

**Actual Behavior**
Page count is displaying as 0. 

**Screenshots**

![Screen Shot 2020-10-14 at 3.39.33 PM.png](https://images.zenhubusercontent.com/5bd7271508843955cc24d3d0/3910dad7-f464-4d89-a50a-ed92fc3f4075)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

**FOR ENGINEERING TEAM ONLY**
Bug Resolution Steps:
 - [ ] Determine why the bug wasn't caught by a test. 
 - [ ] Determine if an automated test needs to fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] Determine if a manual test needs to be fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] If needed, automated test is created.
 - [ ] If needed, manual test is created.
 - [ ] Reason for bug has been documented.
 - [ ] Fix has been deployed to dev environment.
 - [ ] Fix has been deployed to the stage environment.
 - [ ] Bug has been tested in staging (UX or Engineering).

**Comment by rachelschneiderman** [comment]
@klohman 

I am unable to reproduce this locally, dev, and staging. On staging, I believe I found the case where the bug was originally discovered. However, now all documents have the appropriate page counts. I tried filing a document on the case myself, and the page count is correct. I also tried logging in as a docketclerk and a petitionsclerk.

![image.png](https://images.zenhubusercontent.com/5e331dde16551cf7ef3a8f50/1773a169-d8a9-4cbe-8edf-dc7e36517487)
